plugins/connection/incus.py

```diff
@@ -11,8 +11,11 @@
 name: incus
 short_description: Run tasks in Incus instances using the Incus CLI
 description:
-  - Run commands or put/fetch files to an existing Incus instance using Incus CLI.
+  - Run commands or put/fetch files to an existing Incus instances using Incus CLI.
```

Comment:
That looks wrong, the existing wording is correct or it would need to be changed to `to existing Incus instances` instead.

To keep this PR small, I'd just revert to the original description here.